services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/AuthorizationManagerImplUnitTest.java

```diff
@@ -291,6 +298,24 @@ public void testCanAccessActivityPaginationSmallResultSet() throws Exception {
 		assertFalse(canAccess);
 	}
 
+	@Test
+	public void testCanAccessActivityMissingRequired() throws Exception {		 
+		Activity act = new Activity();

```

Comment:
Copy/Paste error :) Might need code for the case with no view scope for the user.

Reply:
Done.